src/View/Helper/FormHelper.php

```diff
@@ -406,6 +406,10 @@ public function create($context = null, array $options = [])
     {
         $append = '';
 
+        if (is_bool($context) || is_string($context)) {
+            deprecationWarning('Using `string` or `bool` for $context is deprecated, use `null` to make a context-less form.');
```

Comment:
A string could be a valid context if someone has a custom form context class for it.

Reply:
I didnt see tests that would match this.

Reply:
But since this is still just a deprecation help for the "fatal error" you get on 4.0 otherwise, even if that use case exists, this should be noted as deprecated, right? As this helps to upgrade to a 4.0 compatible way.

Reply:
I agree that the deprecation is helpful and could help improve someone's upgrade experience.